internal/testutil/tc/advanced_cluster.go

```diff
@@ -3,9 +3,11 @@ package tc
 import (
 	"fmt"
 	"strconv"
+	"strings"
```

Comment:
This file will most likely be deleted during [[TF] Adv Cluster TPF Test Coverage and Clenaup](https://jira.mongodb.org/browse/CLOUDP-286481)